docs/src/quickstart/hello_klio_batch.rst

```diff
@@ -0,0 +1,127 @@
+.. _hello-klio-batch:
+
+Hello Klio Batch Example
+========================
+
+This guide will show you how to set up your development environment to implement Klio,
+create an example Klio batch job and run it on DirectRunner. If you are interested in building
+a streaming Klio job then checkout the :ref:`Klio Streaming Quickstart Guide<hello-klio-streaming>`.
+
+.. attention::
+
+    Be sure to follow the :doc:`installation instructions <installation>` before continuing on.
+
+
+Create a New Klio Batch Job
+---------------------------
+
+:violetemph:`First`, initialize the ``klio_quickstart`` project directory for ``git``:
+
+.. code-block::
+
+    $ git init
+
+.. caution::
+
+    If the |klio-cli|_ was installed via :ref:`option 2 <install-option-2>` or :ref:`option 3 <install-option-3>`, make sure you're ``klio-cli`` virtualenv is activated.
+
+
+:violetemph:`Next`, within your project directory, run the following command:
+
+.. code-block:: sh
+
+    $ klio job create --job-name klio-quick-start --job-type batch --create-resources --use-defaults
+
+
+After responding to the prompts, Klio will:
+
+1. Create a GCS bucket in the provided GCP project you provided in the prompt for output data: ``gs://$GCP_PROJECT-output``.
+2. Create a `Google Stackdriver`_ dashboard in the provided GCP project for you to monitor runtime job metrics.
+3. Create required files within the current working directory.
+
+:violetemph:`Then`, commit the created job files into ``git``:
+
+.. code-block::
+
+    $ git add .
+    $ git commit -m "Initial commit for Klio quickstart example"
+
+
+Run the New Klio Job
+--------------------
+
+.. caution::
+
+    If the |klio-cli|_ was installed via :ref:`option 2 <install-option-2>` or :ref:`option 3 <install-option-3>`, make sure you're ``klio-cli`` virtualenv is activated.
+
+:violetemph:`First`, add a file of IDs elements as text input.
+
+The default event input points to a text file in ``klio-quick-start_input_ids.txt`` in a GCS bucket.
+To get testing quick this file event input should be replaced with a local file.
+A local file containing lines of text corresponding to elements can be created manually
+in the top level job directory for this quickstart. Assuming a linux machine, a local file can
+also be created by running the following command while in the klio job directory.
+
+.. code-block::
+
+    $ { echo hello
```

Comment:
Do you think we need this? Is there a better way to convey this, maybe a screenshot of a file in the directory showing two lines of elements?

I feel like it conveys the message that this is a simple text file with two lines given the name `klio-quick-start_input_ids.txt`.

But this would likely be different across operating systems etc.